src/app/components/categories/categories.component.html

```diff
@@ -0,0 +1,15 @@
+<section id="categories">
+  <div class="categories-category" *ngFor="let category of categories">
+    <a class="categories-header" [id]="category.name + '-header'" routerLink="/categories/{{category.id}}">
+      <h1 [id]="category.name + '-title'">{{category.name}}</h1>
+      <p routerLink="/categories/{{category.id}}">Voir plus</p>
+      <p><strong>></strong></p>
```

Comment:
What is the purpose of those tags ?

Reply:
Yes but why do you wrap it into a span and into a p ? I think you could wrap the span with the "v" inside into the "Voir Plus" p